### PR TITLE
Release 2.3.9

### PR DESCRIPTION
**Pull request by bjhham**

### CODE REVIEW

File: ktor-server/ktor-server-core/api/ktor-server-core.api [hunk 1]
```diff
@@ -763,7 +763,7 @@ public final class io/ktor/server/response/DefaultResponsePushBuilder : io/ktor/
 }
 
 public final class io/ktor/server/response/ResponseCookies {
-	public fun <init> (Lio/ktor/server/response/ApplicationResponse;Z)V
+	public fun <init> (Lio/ktor/server/response/ApplicationResponse;)V
```
Comment (Stexxe) [inline ktor-server/ktor-server-core/api/ktor-server-core.api:766]: Can we put this change in the patch release?

Comment (bjhham) [reply ktor-server/ktor-server-core/api/ktor-server-core.api:766]: I can leave in the unused boolean argument here to maintain compatibility.

**Comment by bjhham** [comment]
Oddly enough, I run into a build error locally that also occurs on 2.3.8...

```
Unable to find a variant of org.jetbrains.kotlin:kotlin-test:1.8.22 providing the requested capability org.jetbrains.kotlin:kotlin-test-framework-junit
```

**Comment by bjhham** [comment]
I've disabled a failing test in `jakarta/TomcatEngineTest` because it also started failing in 2.3.8 https://ktor.teamcity.com/buildConfiguration/Ktor_KtorMatrixCoreLinuxJava11/247880

I have no idea why it started failing today.  Maybe it has something to do with the TeamCity environment, although it's failing locally too.  Strange happenings afoot.

**Review by Stexxe — APPROVED** [review]

**Review by e5l — APPROVED** [review]

**Pull request merged**